docs/administration/segregation.md

```diff
@@ -1,7 +1,118 @@
 # Data segregation
 
-!!! tip "Under construction"
+## Introduction
+
+*Data segregation* in the context of *Cyber Threat Intelligence* refers to the practice of categorizing and separating different types of data or information related to cybersecurity threats based on specific criteria.
+
+This separation helps organizations manage and analyze threat intelligence more effectively and securely and the goal of data segregation is to ensure that only those individuals who are authorized to view a particular set of data have access to that set of data.
+
+## Marking Definitions
+
+### Description
+
+*Marking definitions* are essential in the context of data segregation to ensure that data is appropriately categorized and protected based on its sensitivity or classification level. Marking definitions establish a standardized framework for classifying data.
+
+Marking Definition objects are unique among STIX objects in the [STIX 2.1 standard](https://docs.oasis-open.org/cti/stix/v2.1/stix-v2.1.html) in that they cannot be versioned.  This restriction is in place to prevent the possibility of indirect alterations to the markings associated with a STIX Object.
+
+Multiple markings can be added to the same object, including both object and granular markings. Certain categories of marking definitions or trust groups may enforce rules that specify which markings take precedence over others or how some markings can be added to complement existing ones.
```

Comment:
granular markings ?

Reply:
It will be removed, follow-up our discussion, no need to precise that :)